python/py-natsort/Portfile

```diff
@@ -19,22 +19,30 @@ long_description    When you try to sort a list of strings that \
                     a function `natsorted` that helps sort lists \
                     'naturally', either as real numbers (i.e. \
                     signed/unsigned floats or ints), or as versions.
-homepage            https://github.com/SethMMorton/$realName
+homepage            https://github.com/SethMMorton/${python.rootname}
 
-master_sites        pypi:n/$realName
-distname            $realName-${version}
+master_sites        pypi:n/${python.rootname}
+distname            ${python.rootname}-${version}
 
-checksums           rmd160  7f7ef1bab57862009369c1e97f8b171f8bb17f3f \
-                    sha256  c76ba3e85fba78f276ac06e4d47f2230d1070f9c19413b2a0bfe7de6af311839
+checksums           rmd160  bc57ebaf0994b8c2c5286a2442173dd9506627db \
+                    sha256  ff3effb5618232866de8d26e5af4081a4daa9bb0dfed49ac65170e28e45f2776 \
+                    size    140718
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 if {${subport} ne ${name}} {
-    depends_build       port:py${python.version}-setuptools
-    depends_run         port:py${python.version}-pyicu
-    livecheck.type      none
-} else {
-    livecheck.type      regex
-    livecheck.url       https://pypi.python.org/pypi/$realName
-    livecheck.regex     $realName-(\\d+(?:\\.\\d+)*)${extract.suffix}
+    depends_lib-append \
+                    port:py${python.version}-setuptools
```

Comment:
Normally setuptools only needs to be a build dep. Is it really not the case here ?

Reply:
This change is correct: ```py-setuptools``` is required at runtime as well, since the package installs a ```console_script```.